runtime/ops/worker_host.rs

```diff
@@ -51,8 +51,9 @@ pub struct CreateWebWorkerArgs {
   pub use_deno_namespace: bool,
 }
 
-pub type CreateWebWorkerCb =
-  dyn Fn(CreateWebWorkerArgs) -> WebWorker + Sync + Send;
+pub type CreateWebWorkerCb = dyn Fn(CreateWebWorkerArgs) -> (WebWorker, SendableWebWorkerHandle)
```

Comment:
Could you keep previous signature and expose a method on the worker that could return that handle?

Reply:
No, because SendableWebWorkerHandle is not Clone. The reason I split it out was what it having to be clone greatly complicated all kinds of things.